src/chapters/classes-and-objects/exercises.rst

```diff
@@ -0,0 +1,168 @@
+Exercises: Classes and Objects
+==============================
+
+In these exercises, you will create a ``Robot`` class and use it to create
+four ``Robot`` objects. You will also practice writing functions that display
+information about the objects as well as race them against each other.
+
+.. admonition:: Note
+
+   If your teacher added you to a :ref:`repl.it classroom <replit-classroom-assignments>`
+   or a :ref:`Trinket course <trinket-course-assignments>`, login to your
+   account to access the starter code for each exercise.
+
+   If you are NOT enrolled in a repl.it classroom or Trinket course, use one of
+   the following links to copy the starter code.
+
+   #. `Trinket starter code <https://trinket.io/python/b3c2b6e9ff>`__ (*Remix*
+      before getting started).
+   #. `Repl.it starter code <https://repl.it/@launchcode/LCHS-Classes-Exercises>`__
+      (*Fork* before getting started).
+
+Part 1: Create a New Class
+--------------------------
+
+Open up the starter code and notice that it imports the ``random`` module.
+Below the ``import`` statement, define the ``Robot`` class.
+
+Add Properties to ``Robot``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Inside the class, define the ``__init__`` method. It should include
+   parameters for ``self``, ``name``, ``mass``, and ``year``.
+#. Give a default value to ``year``.
+#. Assign the property values as described in the
+   :ref:`Setting Property Values <set-property-values>` section.
+#. Add one more property called ``distance``. This will NOT have a
+   parameter, and every new ``Robot`` object will start with a ``distance``
```

Comment:
Rephrased.  The idea was to have the students use `self.distance = 0` instead of relying on a parameter value ('self.distance = distance`).

Let me know if the new sentences makes this clear enough.